cli/compose/schema/data/config_schema_v3.3.json

```diff
@@ -89,6 +89,7 @@
           ]
         },
         "container_name": {"type": "string"},
+        "credential_spec": {"type": "string"},
```

Comment:
Does this for with the containers API, or are we going to have to document a special case where fields outside of the deploy section don't work with `docker-compose`?